packages/fluentui/react-builder/src/components/ReaderNarration.tsx

```diff
@@ -4,18 +4,21 @@ import { NarrationComputer, IAriaElement } from './../narration/NarrationCompute
 
 const computer = new NarrationComputer();
 let prevSelector = null;
+let prevVcElement = null;
+let allowVirtualCursor = true;
 let focusableElements = {};
 let elementsPaths = [];
 let selectedElementPath = null;
 let prevNarrationElement = null;
 const aomMissing = !window.hasOwnProperty('getComputedAccessibleNode');
```

Comment:
You don't have to do this PR, but please make sure that you create another PR to make these variables refs in the component with the `useRef` hook https://reactjs.org/docs/hooks-reference.html#useref

Reply:
Good point, so lets leave it for another PR in the future.